tendermint/tests/support/lite/header_tests_failing.json

```diff
@@ -0,0 +1,169 @@
+{
+  "test_cases": [
```

Comment:
The files with the suffix `_failing` currently aren't ran anywhere! These are test-cases that currently fail ... I included them for the sake of transparency and with the goal to make them pass (or understand why they do not apply in Rust-land while they do in Go, if the case).